webhook/forward.go

```diff
@@ -25,7 +25,7 @@ type hookOptions struct {
 	EventTypes []string
 	Repo       string
 	Org        string
-	Port       int
+	Url        string
```

Comment:
Small nit for Go convention:

```suggestion
	URL        string
```